src/interfaces/IOracle.sol

```diff
@@ -5,6 +5,7 @@ pragma solidity >=0.5.0;
 /// @author Morpho Labs
 /// @custom:contact security@morpho.org
 /// @notice Interface that oracles used by Morpho must implement.
+/// @dev It is the responsibility of the oracle to handle stale prices and price manipulations.
```

Comment:
and a lot of other things then, I don't think that it has its place here
```suggestion
```

Reply:
I think it has since we'll have other comments like this in the contest

Reply:
Then we can answer the same thing. Trying to cover the dependencies with more depth is just dangerous for me because we cannot have a 100% coverage and people will ask to be paid for the things that we didn't cover. Still against adding such thing.

Reply:
Yeah but I'm spending my time answering the same questions several times, so why not writing the answer somewhere?

>  is just dangerous 

I don't think it is

Reply:
No but we can find other solutions. We can for example make it clearer that this is not an issue, by acknowledging that nothing more is expected from the oracles and that it's the users responsibility to select market with oracles that are safe. Without ever precising what "safe" means.